README.md

```diff
@@ -107,11 +107,11 @@ Read [INSTALL](INSTALL) file for a general summary about dependencies and
 configure options. Read [build.md](scripts/build.md) for the comprehensive
 compilation guide.
 
-    $ git clone https://github.com/dreamer/dosbox-staging.git
-    $ cd dosbox-staging
-    $ ./autogen.sh
-    $ ./configure
-    $ make
+    git clone https://github.com/dreamer/dosbox-staging.git
+    cd dosbox-staging
+    ./autogen.sh
+    ./configure
+    make
```

Comment:
Maybe shell syntax highlight? :)

Reply:
Will do - but I do like how tidy the raw text looked in your original version :-)
The linter didn't like the $ prefixes (justified by the them making it harder to copy and paste).

Reply:
Fenced, along with a couple others.